src/common/Graylog.cc

```diff
@@ -24,7 +24,7 @@ Graylog::Graylog(const SubsystemMap * const s, std::string logger)
       m_log_dst_valid(false),
       m_hostname(""),
       m_fsid(""),
-      m_logger(logger),
+      m_logger(std::move(logger)),
```

Comment:
> logger is not a rvalue _reference_

i'm not sure what you're saying here. `m_logger(logger)` does call the `std::string` copy ctor, and adding `std::move()` causes it to use the move ctor instead

your point remains that this is only called once on initialization, so it won't ever show up in the profiler. but a copy is a copy, and I'd prefer to move when it's appropriate